Showcase/Showcase/Pages/Tokens/Color/NamedColor/NamedColor+Action.swift

```diff
@@ -0,0 +1,192 @@
+            case .colorActionOnBackgroundActionPrimaryHover:
+                return theme.colorActionOnBackgroundActionPrimaryHover
+            case .colorActionOnBackgroundActionPrimaryHoverOnBackgroundEmphasized:
+                return theme.colorActionOnBackgroundActionPrimaryHoverOnBackgroundEmphasized
+            case .colorActionOnBackgroundActionPrimaryPressed:
+                return theme.colorActionOnBackgroundActionPrimaryPressed
+            case .colorActionOnBackgroundActionPrimaryPressedOnBackgroundEmphasized:
+                return theme.colorActionOnBackgroundActionPrimaryPressedOnBackgroundEmphasized
+            case .colorActionOnBackgroundActionPrimaryLoading:
+                return theme.colorActionOnBackgroundActionPrimaryLoading
+            case .colorActionOnBackgroundActionPrimaryLoadingOnBackgroundEmphasized:
+                return theme.colorActionOnBackgroundActionPrimaryLoadingOnBackgroundEmphasized
+            case .colorActionOnBackgroundActionPrimaryFocus:
+                return theme.colorActionOnBackgroundActionPrimaryFocus
+            case .colorActionOnBackgroundActionPrimaryFocusOnBackgroundEmphasized:
+                return theme.colorActionOnBackgroundActionPrimaryFocusOnBackgroundEmphasized
+//            case .colorActionOnBackgroundActionSecondaryEnabled:
+//                return theme.colorActionOnBackgroundActionSecondaryEnabled
+//            case .colorActionOnBackgroundActionSecondaryEnabledOnBackgroundEmphasized:
+//                return theme.colorActionOnBackgroundActionSecondaryEnabledOnBackgroundEmphasized
+//            case .colorActionOnBackgroundActionSecondaryHover:
+//                return theme.colorActionOnBackgroundActionSecondaryHover
+//            case .colorActionOnBackgroundActionSecondaryHoverOnBackgroundEmphasized:
+//                return theme.colorActionOnBackgroundActionSecondaryHoverOnBackgroundEmphasized
+//            case .colorActionOnBackgroundActionSecondaryPressed:
+//                return theme.colorActionOnBackgroundActionSecondaryPressed
+//            case .colorActionOnBackgroundActionSecondaryPressedOnBackgroundEmphasized:
+//                return theme.colorActionOnBackgroundActionSecondaryPressedOnBackgroundEmphasized
+//            case .colorActionOnBackgroundActionSecondaryLoading:
+//                return theme.colorActionOnBackgroundActionSecondaryLoading
+//            case .colorActionOnBackgroundActionSecondaryLoadingOnBackgroundEmphasized:
+//                return theme.colorActionOnBackgroundActionSecondaryLoadingOnBackgroundEmphasized
+//            case .colorActionOnBackgroundActionSecondaryFocus:
+//                return theme.colorActionOnBackgroundActionSecondaryFocus
+//            case .colorActionOnBackgroundActionSecondaryFocusOnBackgroundEmphasized:
+//                return theme.colorActionOnBackgroundActionSecondaryFocusOnBackgroundEmphasized
+            }
+        }
+    }
+}
```

Comment:
Normalement tu dois avoir des warnings tels que :
- "Blanket Disable Command Violation: The disabled 'identifier_name' rule should be re-enabled before the end of the file (blanket_disable_command)"
- Blanket Disable Command Violation: The disabled 'line_length' rule should be re-enabled before the end of the file (blanket_disable_command)
- "Blanket Disable Command Violation: The disabled 'function_body_length' rule should be re-enabled before the end of the file (blanket_disable_command)"

Les warnings désactivés dans un fichier doivent être réactivés ensuite.

Il faut ajouter à la fin du fichier, après un saut de ligne, les lignes : 

```swift
// swiftlint:enable identifier_name
// swiftlint:enable line_length
// swiftlint:enable function_body_length
```